examples/howto/Model_Builder.myst.md

```diff
@@ -0,0 +1,251 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Using ModelBuilder class for deploying PyMC models 
+:::post: Sep 12, 2022:tags: model builder class, model class, linear model:category: Beginner:author: Shashank Kirtania, Thomas Wiecki:::
```

Comment:
This also needs to follow the right formatting (see https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell), each metadata in its own line, the ending colon fence also on a line of its own, brackets around `post` in the directive definition.